community/cypher/cypher-compiler-2.3/src/main/scala/org/neo4j/cypher/internal/compiler/v2_3/codegen/CodeGenerator.scala

```diff
@@ -82,13 +81,7 @@ object CodeGenerator {
   def n = System.lineSeparator()
 
   private def generateCodeFromInstructions(className: String, instructions: Seq[Instruction]) = {
-    val importLines: Set[String] =
-      instructions.
-        map(_.importedClasses()).
-        reduceOption(_ ++ _).
-        getOrElse(Set.empty)
-
-
+    val importLines = instructions.map(_.allImportedClasses).reduceOption(_ ++ _).getOrElse(Set.empty)
```

Comment:
Well, the reason for the change is to make this line more consistent with the bunch of similar lines below, which are all one-liners. IMO, if this is supposed to be multi-line, we should make the others that too. Would you prefer that?